packages/hosted-assets/package.json

```diff
@@ -15,7 +15,8 @@
   "main": "dist",
   "files": [
     "dist",
-    "_index.scss"
+    "_index.scss",
+    "**/*.elm"
```

Comment:
Only typescript and the scss file was being published. This should fix the elm files not being published.